docs/root/start/sandboxes/cache.rst

```diff
@@ -0,0 +1,289 @@
+and serve it to subsequent requests. In this demo, the responses that are served by the deployed services are stored in :repo:`/examples/cache/responses.yaml`. 
+This file is mounted to both services' containers, so any changes made to the stored responses while the services are running should be instantly effective (no need to rebuild or rerun). 
+For the purposes of the demo, a response's date of creation is appended to its body before being served. 
+An Etag is computed for every response for validation purposes, which only depends on the response body in the yaml file (i.e. the appended date is not taken into account). 
+Cached responses can be identified by having an ``age`` header. Validated responses can be identified by having a generation date older than the ``date`` header;
+as when a response is validated the ``date`` header is updated, while the body stays the same.
+
+Running the Sandbox
+~~~~~~~~~~~~~~~~~~~
+
+The following documentation runs through the setup of an Envoy cluster organized
+as is described in the image above.
+
+**Step 1: Install Docker**
+
+Ensure that you have a recent versions of ``docker`` and ``docker-compose`` installed.
+
+A simple way to achieve this is via the `Docker Desktop <https://www.docker.com/products/docker-desktop>`_.
+
+**Step 2: Clone the Envoy repo**
+
+If you have not cloned the Envoy repo, clone it with:
+
+``git clone git@github.com:envoyproxy/envoy``
+
+or
+
+``git clone https://github.com/envoyproxy/envoy.git``
+
+**Step 3: Start all of our containers**
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/cache
+    $ docker-compose build --pull
+    $ docker-compose up -d
+    $ docker-compose ps
+
+           Name                      Command            State                             Ports                          
+    ------------------------------------------------------------------------------------------------------------------------
+    cache_front-envoy_1   /docker-entrypoint.sh /bin ... Up      10000/tcp, 0.0.0.0:8000->8000/tcp, 0.0.0.0:8001->8001/tcp
+    cache_service1_1      /bin/sh -c /usr/local/bin/ ... Up      10000/tcp, 8000/tcp                                      
+    cache_service2_1      /bin/sh -c /usr/local/bin/ ... Up      10000/tcp, 8000/tcp    
+
+**Step 4: Test Envoy's HTTP caching capabilities**
+
+You can now send a request to both services via the ``front-envoy``. Note that since the two services have different routes,
+identical requests to different services have different cache entries (i.e. a request sent to service 2 will not be served by a cached
+response produced by service 1).
+
+To send a request: 
+
+``curl -v localhost:8000/service/<service_no>/<response>``
+
+``service_no``: The service to send the request to, 1 or 2.
+
+``response``: The response that is being requested. The responses are found in :repo:`/examples/cache/responses.yaml`.
```

Comment:
it  would be quite helpful to list these here

Reply:
Do you mean just their names and descriptions, or the whole yaml file?
The examples in step 4 describe each one of them with its usage and explanation. Also, the yaml file is meant to be easily editable so that the users can try their own responses, existing responses are just examples.

Reply:
i just meant perhaps in brackets (`valid-for-minute`, `private`, or `no-cache`)

Reply:
> users can try their own responses

yep been playing with changing the file...

Reply:
I updated the docs to clarify example responses.